Add Nitropage

0x1d opened the pull request:
This change add the Nitropage-Starter project deployment. 

Nitropage is a completely free, open-source, self-hostable alternative to premium, visual drag-and-drop website builders such as Squarespace and Wix.
Visit https://nitropage.com/ for more information.

0x1d commented on nitropage/deploy.yaml:
```diff
@@ -0,0 +1,47 @@
+          - global: true
+    env:
+      # Path to database file
+      - DATABASE_URL=file:/var/db/dev.db
```
yes it was intended, as i first thought this deployment should serve as a demo - but the more i think about it, i could as well add persistence. I will update the deployment accordingly.

0x1d replied:
i've just added persistence

Dimokus88 commented:
There is also no response from the creator of PR.
I suggest rejecting/closing.
@arno01 @anilmurty

katywings commented:
Quick info 😅, I am in personal contact with @0x1d and he will respond to you very soon! Thank you for your patience ☺️.

0x1d commented:
Hi @Dimokus88 
Sorry, i didn't see you comment. 
I've updated the PR and added the Dockerfile, scripts and updated to the latest Nitropage version.
Hope now everything is in order.
Cheers!

Dimokus88 commented:
Thx @katywings !
Awesome @0x1d!
Ready to merge @brewsterdrinkwater @arno01 .

0x1d commented:
Thank you 💪
